scripts.js

```diff
@@ -22,12 +23,52 @@ const string = prompt('Enter a string: ');
 const result = reverseString(string);
 console.log(result);
```

Comment:
These two lines can be combined into one by `console.log(reverseString(string))`